README.md

```diff
@@ -19,15 +19,51 @@ A modular vector database interface supporting multiple backends (Weaviate, Milv
 
 ### Installation
 
+First, clone the repository and navigate into the directory:
+
 ```bash
-# Clone the repository
-git clone <repository-url>
+git clone [https://github.com/your-username/maestro-knowledge.git](https://github.com/your-username/maestro-knowledge.git)
 cd maestro-knowledge
+```
+
+You will need Python 3.11+ and `uv`. You can install `uv` with:
+```bash
+pip install uv
+```
```

Comment:
uv should just be listed as a prerequisite rather than giving an install step since there are many ways to install it